### PR TITLE
Congruencies v1

### PR DESCRIPTION
**Pull request by alimalim77**
## Description
Congruences is a new concept addition to Discrete Math. It focuses on the logic behind the division and remainder portion of mathematics. Congruences are also used to deduce properties like Reflexive, Symmetric and Transitive.

## Type of Change
- Closes Issue -> https://github.com/Codecademy/docs/issues/2390.
- Created a new concept entry for Discrete Math Proofs.
-  Used LaTeX to Markdown conversion using '$$' by placing it before and after the mathematical expression. 
- Attached Explanation and Example wherever required.

## Snippet
image
image
image

## Checklist
- [✓] All writings are my own.
- [✓ ] My entry follows the Codecademy Docs style guide.
- [✓] My changes generate no new warnings.
- [✓] I have performed a self-review of my own writing and code.
- [✓] I have checked my entry and corrected any misspellings.
- [✓] I have made corresponding changes to the documentation if needed.
- [✓] I have confirmed my changes are not being pushed from my forked main branch.
- [✓] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [✓] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes # in the "Description" above).

### CODE REVIEW

**Comment by alimalim77** [comment]
Hello maintainers, @yangc95 @SSwiniarski @caupolicandiaz 
Please review the PR if possible. Thanks!

**Comment by alimalim77** [comment]
Any updates?

**Comment by SSwiniarski** [comment]
@alimalim77, @yangc95 has this marked as currently under review.

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:21]: The preview for this section seems a bit off. Can we replace $$\neq$$ with an alternative symbol or text

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:30]: Here as well (see preview)

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:34]:
```suggestion
If _a_ and _b_ are integers and _m_ is a positive integer, then _a_ is congruent to _b_ modulo _m_ if _m|(a − b)_. It is termed as Congruence Relation.
```

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:18]: Doesn't matter much to me if we use `n` (traditional) `m` or any other character to represent the integer so long as it is consistent throughout this. I made everything _m_ but if there's one that I missed feel free to self-revise

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [reply content/discrete-math/concepts/congruences/congruences.md:21]: Also, it may be nice to preface these headers based on their meaning.

Can we also do the same for the bottom portion that goes into the properties? So create a h2 named something like "Congruence Properties" and have a short summary of what the different properties entail before going into linear congruences, reflexive, etc.. (also turn those into h3s)

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:52]:
```suggestion
### Linear Congruences
```

Turn the others below into h3s as well

**Review by yangc95 — CHANGES_REQUESTED** [review]
Hey @alimalim77 nice job at tackling this, I made some direct edits and also had additional comments to improve clarity and pedagogy in explaining this concept.

I found Wolfram to be a good resource to brush up on understanding this concept: https://mathworld.wolfram.com/Congruence.html

Please let me know if you need any assistance and I can take a deeper dive with you. Thanks!

File: content/discrete-math/concepts/congruences/congruences.md
Comment (alimalim77) [reply content/discrete-math/concepts/congruences/congruences.md:18]: Hello, I just checked and all the integers are represented by m. Thanks.

File: content/discrete-math/concepts/congruences/congruences.md
Comment (alimalim77) [reply content/discrete-math/concepts/congruences/congruences.md]: Noted. Thanks!

File: content/discrete-math/concepts/congruences/congruences.md
Comment (alimalim77) [reply content/discrete-math/concepts/congruences/congruences.md:30]: I get it but that is standard notation to denote the non-divisibility. Replacing it with some other operator / character might make the statement vague.

File: content/discrete-math/concepts/congruences/congruences.md
Comment (yangc95) [inline content/discrete-math/concepts/congruences/congruences.md:23]:
```suggestion
If _a_ and _b_ are integers with a ≠ 0 then _a_ divides _b_ if an integer _c_, exists given that _b = ac_.
```

Comment (yangc95) [reply content/discrete-math/concepts/congruences/congruences.md:23]: I wonder if just using the symbol will work

Comment (alimalim77) [reply content/discrete-math/concepts/congruences/congruences.md:23]: Sure. It forgot that it previews here then it should in case of preview as well. Making the changes.

**Review by yangc95 — APPROVED** [review]
Passing this onto @KTom101 for a second review!

**Review by KTom101 — CHANGES_REQUESTED** [review]
@alimalim77 This looks good. Some italics were missing, please add them for consistency.

File: content/discrete-math/concepts/congruences/congruences.md
Comment (alimalim77) [reply content/discrete-math/concepts/congruences/congruences.md]: Resolved

**Review by KTom101 — APPROVED** [review]
@alimalim77 Thanks for your patience. I will merge this.

**Pull request merged**
